castai/resource_castai_gke_cluster.go

```diff
@@ -19,6 +19,7 @@ const (
 	FieldGKEClusterName                    = "name"
 	FieldGKEClusterProjectId               = "project_id"
 	FieldGKEClusterRegion                  = "region"
```

Comment:
If location is region or region+zone, then region I believe can be made redundant

Reply:
We can keep only location field, but compute region from location since we need it for api
```
{
    "clusterName": "string",
    "location": "string",
    "projectId": "string",
    "region": "string"
  }
  ```
  
  Also you can check if we really need to send region.

Reply:
A region removed from the provider but it is still required by our API